src/test/java/gov/healthit/chpl/aqa/pageObjects/SurveillanceComplaintsPage.java

```diff
@@ -0,0 +1,117 @@
+package gov.healthit.chpl.aqa.pageObjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Class SurveillanceComplaintsPage definition.
+ */
+public final class SurveillanceComplaintsPage extends BasePage {
+    private SurveillanceComplaintsPage() {}
+
+    public static WebElement actions(final WebDriver driver) {
+        return driver.findElement(By.id("actions"));
+    }
+
+    public static WebElement addNewComplaintButton(final WebDriver driver) {
+        return driver.findElement(By.id("add-new-complaint"));
+    }
+
+    public static WebElement addComplaintFormTitle(final WebDriver driver) {
+        return driver.findElement(By.xpath("//*[@id=\"chpl-complaint\"]/div[1]/h2"));
+    }
+
+    public static WebElement associatedCriteriaValue(final WebDriver driver) {
+        return driver.findElement(By.xpath("//*[@id=\"chpl-complaint\"]/div[13]/div[1]/span/button"));
```

Comment:
One of the comments on this ticket has some rearrangement requests. My guess is that the `div[13]` is not going to work for long. Maybe each of the form elements could have an ID that you could start from? Or the button should...

Reply:
Requested locators